server/TypeDBServer.java

```diff
@@ -156,7 +159,7 @@ private void configureTracing() {
         typeDBService = new TypeDBService(typedb);
         MigratorService migratorService = new MigratorService(typedb, Version.VERSION);
 
-        return NettyServerBuilder.forPort(configuration.port())
+        return NettyServerBuilder.forAddress(configuration.server().address())
```

Comment:
we now newly support listening to all addresses (using `0.0.0.0`) as well as just local address (`localhost`)